packages/react-components/src/components/TagGroup/TagGroup.css

```diff
@@ -11,5 +11,13 @@
 }
 
 .bcds-react-aria-TagGroup--Text-error {
+  display: flex;
   color: var(--typography-color-danger);
 }
+
+.bcds-react-aria-TagGroup--Text-error > svg {
+  padding-right: var(--layout-padding-xsmall);
+  color: var(--icons-color-danger);
+  width: var(--icons-size-medium);
```

Comment:
Inside of a flex container, `width` has potential to collapse depending on the other children of the container. Here's what happens with a long error message:

<img width="1840" alt="TagGroup with icon using width" src="https://github.com/user-attachments/assets/733d1c61-9ca0-4858-943f-c11971ca4dcf">

If you use `min-width` instead and keep the `height` as it is, you should get what you're looking for:

<img width="1840" alt="TagGroup with icon using min-width" src="https://github.com/user-attachments/assets/a1559f6b-1d90-4ec1-8ea0-9178f9cfd813">

Reply:
Fixed in [883c458](https://github.com/bcgov/design-system/pull/429/commits/883c458cc2a17a04845b882a7ea3a0eb436d6e6b), flagging that we might need to refactor this on a couple of other components too!